CMakeLists.txt

```diff
@@ -37,6 +37,10 @@ list(GET version_numbers 0 VERSION_MAJOR)
 list(GET version_numbers 1 VERSION_MINOR)
 set(VERSION_MAJ_MIN "${VERSION_MAJOR}.${VERSION_MINOR}")
 
+if (NOT VAST_LOG_LEVEL)
+  set(VAST_LOG_LEVEL "-1")
```

Comment:
Would you mind elaborating why you set the log level to a string an not a numeric value?

Reply:
cmake does not have an integral type. There is no semantic difference between `-1` and `"-1"`. I added the quotation marks to not confuse the reader about this.